h2o-core/src/main/java/water/JettyHTTPD.java

```diff
@@ -305,7 +305,13 @@ public void registerHandlers(HandlerWrapper handlerWrapper) {
     ServletContextHandler context = new ServletContextHandler(
         ServletContextHandler.SECURITY | ServletContextHandler.SESSIONS
     );
-    context.setContextPath("/");
+
+    if(null != H2O.ARGS.context_path && !H2O.ARGS.context_path.isEmpty()) {
+      context.setContextPath(H2O.ARGS.context_path);
```

Comment:
I see, so we want both `conext_path H2O` and `conext_path /H2O` to have the same effect?